schema_v2.csql

```diff
@@ -6,7 +6,23 @@ CREATE TABLE effective_rules (
   rule_id   uuid PRIMARY KEY,
   country   text,
   region    text,
+  party     text,
   timezone  text,
   starts    timestamp,
   ends      timestamp
+);
+
+CREATE TABLE when_keys (
+  id        uuid,
+  section   text,
+  key       text,
+  PRIMARY KEY(section, key)
+);
+
+CREATE TABLE whens (
```

Comment:
Change rule_id to "string". It will change its format soon. For now, you can still store UUIDs in there.